doc/production.rst

```diff
@@ -4,6 +4,15 @@
 Production
 **********
 
+Unlike the Development environment, that uses Vagrant pre-configured virtual
```

Comment:
In development.rst we wrote "development environment", to be coherent we should use the same criteria.